packages/front-end/components/Experiment/HashVersionSelector.tsx

```diff
@@ -42,33 +49,66 @@ export function HashVersionTooltip({ children }: { children: ReactNode }) {
 export default function HashVersionSelector({
   value,
   onChange,
+  project,
 }: {
   value: 1 | 2;
   onChange: (value: 1 | 2) => void;
+  project?: string;
 }) {
+  const { data: sdkConnectionsData } = useSDKConnections();
+  const hasSDKWithNoBucketingV2 = !allConnectionsSupportBucketingV2(
+    sdkConnectionsData?.connections,
+    project
+  );
+
   return (
-    <SelectField
-      label="Hashing Algorithm"
-      options={[
-        { label: "V1 (Legacy)", value: "1" },
-        { label: "V2", value: "2" },
-      ]}
-      value={value + ""}
-      onChange={(v) => {
-        onChange((parseInt(v) || 2) as 1 | 2);
-      }}
-      helpText={
-        <>
-          V2 fixes potential bias issues when using similarly named tracking
-          keys, but is only supported in{" "}
-          <HashVersionTooltip>
-            <span className="text-primary">
-              some SDK versions <FaQuestionCircle />
-            </span>
-          </HashVersionTooltip>
-          .
-        </>
-      }
-    />
+    <>
+      <SelectField
+        label="Hashing Algorithm"
+        options={[
+          { label: "V1 (Legacy)", value: "1" },
+          { label: "V2", value: "2" },
+        ]}
+        value={value + ""}
+        onChange={(v) => {
+          onChange((parseInt(v) || 2) as 1 | 2);
+        }}
+        helpText={
+          <>
+            V2 fixes potential bias issues when using similarly named tracking
+            keys, but is only supported in{" "}
+            <HashVersionTooltip>
+              <span className="text-primary">
+                some SDK versions <FaQuestionCircle />
+              </span>
+            </HashVersionTooltip>
+            .
+          </>
+        }
+      />
+
+      {hasSDKWithNoBucketingV2 && (
+        <div className="mt-2 alert alert-warning">
+          <FaExclamationTriangle className="mr-1" />
+          Some of your{" "}
+          <a href="/sdks" target="_blank">
+            SDK Connections <FaExternalLinkAlt />
+          </a>{" "}
```

Comment:
Is this link actionable?  How are they supposed to know what to do when they land on this page?  Would be better to maybe link to a docs page instead that explains the issue and tells people how to update.

Reply:
removed the link. discussed later maybe including a `?capabilities` query string that would highlight connections with issues